configs/xcache/config.d/40-osg-monitoring.cfg

```diff
@@ -9,6 +9,8 @@
 # https://opensciencegrid.org/docs/data/stashcache/overview/
 
 
+all.sitename $(resourcename)
```

Comment:
This is where it gets dicey. Should we add a comment that this should match somethign in topology or not really?

Reply:
This is what we have in `10-common-site-local.cfg`:

```
# The server name to use for monitoring and reporting; this should
# match the registered resource name in topology.opensciencegrid.org.
# See https://github.com/opensciencegrid/topology/#topology
#
# set resourcename = OSG_RESOURCE_NAME
```

How's this look?

Reply:
that looks good